app/GraphQL/Subscription/Mutations/Plans/PlanMutation.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\GraphQL\Subscriptions\Mutations\Plans;
+
+use Kanvas\Subscription\Plans\Actions\CreatePlan;
+use Kanvas\Subscription\Plans\Actions\UpdatePlan;
+use Kanvas\Subscription\Plans\Repositories\PlanRepository;
+use Kanvas\Subscription\Plans\DataTransferObject\Plan as PlanDto;
+use Kanvas\Subscription\Plans\Models\Plan as PlanModel;
+use Kanvas\Apps\Models\Apps;
+use Illuminate\Support\Facades\Auth;
+use Stripe\Stripe;
+use Stripe\Product as StripeProduct;
+
+class PlanMutation
+{
+    public function __construct()
+    {
+        $app = app(Apps::class);
+        Stripe::setApiKey($app->get('stripe_secret'));
+    }
+
+    /**
+     * create.
+     *
+     * @param  mixed $root
+     * @param  array $req
+     *
+     * @return PlanModel
+     */
+    public function create(array $req): PlanModel
+    {
+        $app = Apps::findOrFail($req['input']['apps_id']);
```

Comment:
@Keysie27 dont accept app id from param use always the app DI, so in this case $this->App